css_docs/button.css

```diff
@@ -1 +1,334 @@
-@import url('/utility.css');
\ No newline at end of file
+@import url('/utility.css');
+@import url('./css_docs/colors.css');
+@import url('./css_docs/Typography.css');
+@import url('./css_docs/Reset.css');
+
+/* Primary */
+
+.primary-btn{
+    padding: 0.5rem 1rem;
+  font-size: 1rem;
+  margin: 0.5rem 1rem;
```

Comment:
Try to use global variables for properties like padding and margin 
--space-xsmall:0.5rem
--space-small:0.75rem 
You can add for commonly used values